deploy/images.libsonnet

```diff
@@ -0,0 +1,15 @@
+// last updated: 1/28/2019
+{
+    _images+:: {
+        alertmanager: "quay.io/cortexproject/alertmanager:master-0ef1ce95",
```

Comment:
Would it be possible to pull the image hashes from `_config`?

Reply:
sure. though there is something nice about keeping images all in one area, so you can glance at all the various versions running for other services such as memcached, postgres, consul, etc.

I guess we could do `$._config.images.alertmanager` instead of putting the image field within `$._config.alertmanager.image` for example. What do you think?

Reply:
These templates need to be updated to use the single binary. When this is done, I think we can easily roll this up into `_config` since there will only be one image